apps/eec/fields/index.js

```diff
@@ -124,6 +135,26 @@ module.exports = {
       value: 'problem-valid-until'
     }
   },
+  'detail-signin-email': {
+    mixin: 'input-text',
+    validate: [
```

Comment:
should this field be `required`?

Reply:
Yes, UCD decided all these fields should be required if selected